### PR TITLE
Remove topology header

### PR DESCRIPTION
**Pull request by ChuckHastings**
Remove the header file `topology.cuh` which is no longer used.

### CODE REVIEW

File: cpp/src/components/connectivity.cu [hunk 1]
```diff
@@ -27,8 +27,6 @@
 #include <type_traits>
 #include <utilities/graph_utils.cuh>
```
Comment (seunghwak) [inline cpp/src/components/connectivity.cu:28]: May better update this to

```
#include <utilities/graph_utils.cuh>

#include <type_traits>
```

to be consistent with the rest of the cuGraph.

**Review by seunghwak — APPROVED** [review]
LGTM (I added a comment to clean-up one more our code debt).

File: cpp/src/components/connectivity.cu (hunk 1 above)
Comment (ChuckHastings) [reply cpp/src/components/connectivity.cu:28]: Reorganized includes.

**Review by rlratzel — APPROVED** [review]

**Comment by ChuckHastings** [comment]
@gpucibot merge

**Pull request merged**